src/data/quickstarts.json

```diff
@@ -3118,6 +3120,7 @@
         "url": "https://docs.newrelic.com/docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-dropwizard-integration"
       }
     ],
+    "keywords": ["featured"],
```

Comment:
Did we manually set this to test? or are these actually set at the QS level now?

Reply:
I manually set this to test for the moment